Frontend/ui-driver/src/Flow.purs

```diff
@@ -1135,6 +1135,7 @@ homeScreenFlow = do
           modifyScreenState $ HomeScreenStateType (\homeScreen -> homeScreen{ props {enterOtpModal = false, showDottedRoute = true}, data{ route = [], activeRide{status = INPROGRESS}}})
           void $ lift $ lift $ toggleLoader false
           _ <- updateStage $ HomeScreenStage RideStarted
+          void $ pure $ openNavigation 0.0 0.0 updatedState.data.activeRide.dest_lat updatedState.data.activeRide.dest_lon
```

Comment:
Before we open the navigation, I think it was discussed to have a 2 seconds delay here

Reply:
Can you check and confirm once?

Reply:
Sure